pages/_middleware.ts

```diff
@@ -38,14 +38,27 @@ export default async function middleware(req : NextRequest) : Promise<NextRespon
       }
     }
 
+    const lettersToCheck = WORD.split("")
+    const match = word.split("").map((letter) => (
+      {
+        letter: letter,
+        score: "bad"
+      }
+    ))
+    word.split("").forEach((letter, i) => {
+      if (WORD[i] === letter) {
+        match[i].score = "good"
+        lettersToCheck.splice(i, 1)
+      }
+    })
+    word.split("").forEach((letter, i) => {
+      if (lettersToCheck.includes(letter) && match[i].score !== "good") {
+        match[i].score = "off"
```

Comment:
```suggestion
        match[i].score = "off"
        lettersToCheck.splice(lettersToCheck.indexOf(letter), 1)
```

Reply:
@rauchg this line needs to be added for that fix to work, sorry didn't catch it in time. want me to open another pr or do you want to commit it directly?